Solutions/PaloAltoCDL/Hunting Queries/PaloAltoCDLOutdatedConfigVersions.yaml

```diff
@@ -0,0 +1,26 @@
+id: 555bf415-e171-4ad2-920f-1a4a96a9644c
+name: PaloAlto - Outdated config vesions
+description: |
+  'Query shows outdated config vesions'

```

Comment:
Should the query description be the Query shows agent versions. I do not see the below being used in the query for comparison.
let cur_ver = dynamic(['0.1']); //put latest config version here